pca/db/database_model.py

```diff
@@ -417,3 +419,38 @@ def save(self, *args, **kwargs):
         else:
             self.internal_ip_int = None  # IPv6 integer representations are too large to store in MongoDB currently
         super(ApplicationDoc, self).save(*args, **kwargs)
+
+class UserReportDoc(RootDoc):
+    """Provides schema for User Report doc.
+    customer - a customer ID that corresponds to a CustomerDoc
+    assessment - an assessment ID that corresponds to an AssessmentDoc
+    campaign - a campaign ID that corresponds to a CampaignDoc
+    first_report - the first time a user responds to a phishing email. Optional, may not be available from GoPhish
+    total_num_reports - the total number of responses (reports) for a campaign.
```

Comment:
How about this?  I don't think you need the "Optional..." bit, since that is covered by the field definition below.
```suggestion
    first_report - the timestamp when the first user click ("report") was received during a phishing campaign
    total_num_reports - the total number of user clicks ("reports") for a campaign
```

Reply:
Changes applied in 76ffea3eb11ba5c77d2593759504e71da3353c4e